go/oasis-node/cmd/registry/runtime/runtime.go

```diff
@@ -372,7 +372,7 @@ func runtimeFromFlags() (*registry.Runtime, signature.Signer, error) { // nolint
 			GroupSize:         viper.GetUint64(CfgExecutorGroupSize),
 			GroupBackupSize:   viper.GetUint64(CfgExecutorGroupBackupSize),
 			AllowedStragglers: viper.GetUint64(CfgExecutorAllowedStragglers),
-			RoundTimeout:      viper.GetDuration(CfgExecutorRoundTimeout),
+			RoundTimeout:      viper.GetInt64(CfgExecutorRoundTimeout),
```

Comment:
not related to this PR but i also just noticed that we don't verify any (other than `GroupSize`) of the `ExecutorParameters` when registering a runtime.

Reply:
Specifically for `RoundTimeout` should probably allow only `0<=X<=max`, `max` mostly so we don't need to worry about overflowing. Maybe also not allow non sensible small values such us `<5`?

Reply:
Validation for RoundTimeout (actually exactly for "<5" :grin:) has already been added in this PR?

Reply:
Oh, sorry missed that :-) maybe also impose a sensible max limit? (and add validation for other executor params that miss it / or we can open an issue for the rest)

Reply:
Refactored the runtime descriptor validity checks a bit, please take another look.

Reply:
looks good :+1: